packages/plugin-connector/src/data-sources/index.js

```diff
@@ -66,14 +69,19 @@ const generateSourceEnvironmentVariables = (sourceName, sourceConfig) => {
 	 * @param {any} obj
 	 */
 	const generateNestedEnvVars = (obj, currentKey = '') => {
-		for (const [key, value] of Object.entries(obj)) {
-			const newKey = currentKey ? `${currentKey}__${key}` : key;
-
-			if (typeof value === 'object') {
-				generateNestedEnvVars(value, newKey);
-			} else {
-				sourceEnvVars[`EVIDENCE_SOURCE__${sourceName}__${newKey}`] = value.toString();
+		try {
+			for (const [key, value] of Object.entries(obj)) {
+				const newKey = currentKey ? `${currentKey}__${key}` : key;
+
+				if (typeof value === 'object') {
+					generateNestedEnvVars(value, newKey);
+				} else {
+					sourceEnvVars[`EVIDENCE_SOURCE__${sourceName}__${newKey}`] = value.toString();
+				}
 			}
+		} catch (e) {
+			console.log({ obj, currentKey });
```

Comment:
`console.error` ? Instead?

Reply:
I don't know if this is needed anymore at all